templates/articles/focused_article_page.html

```diff
@@ -12,15 +12,17 @@
                 {% include 'includes/generic-intro.html' with breadcrumb=True title=page.title intro=page.intro classes="generic-intro--dark" %}
             {% else %}
                 {% include 'includes/generic-intro.html' with breadcrumb=True title=page.title %}
-            {% endif %}    
-            {% if page.author_name %}
+            {% endif %}
+            {% if page.authors %}
                 <div class="generic-intro__meta">
                     <div class="tna-container">
                         <div class="tna-column tna-column--full">
                             <ul class="tna-ul tna-ul--plain">
-                                {% if page.author %}
-                                    <li>By <a href="{% pageurl page.author %}" rel="author" data-component-name="Page header" data-link-type="Link" data-link="{{ page.author_name }}">{{ page.author_name }}</a></li>                    
-                                {% endif %}
+                                <li>By 
+                                    {% for author in page.authors %}
+                                        <a href="{% pageurl author.author %}" rel="author" data-component-name="Page header" data-link-type="Link" data-link="{{ author.author.title }}">{{ author.author.title }}</a>{% if not forloop.last %}{% if forloop.counter == page.authors|length|add:"-1" %} and {% else %}, {% endif %}{% endif %}
```

Comment:
Can the 'if not foorloop' go on the next line - it is very long as it stands.

Reply:
I'm going to leave it as is for now if that's okay - I tried putting it on another line and formatting it a bit nicer but then it messed up the spacing of the displayed text, but if/when we get a HTML formatter set up, this'll be changed then 😀